starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -62,6 +64,8 @@ public Inventory create(Inventory inventory) {
    */
   public Optional<Inventory> retrieve(String id) {
     // TODO
+    List<Inventory> storage = findAll();
```

Comment:
Here I think you forgot to return the value of the findall(). This always returns an empty optional.

Reply:
Actually, I don't think this should be edited at all for this task, so can you revert this change for this branch?